### DISCUSSION TITLE
跑作物的时候，输出大量日值LAI=0.05

### DISCUSSION THREAD

**Discussion by 12wlj** [NCAR/hrldas#191] [Q&A]
跑作物的时候，输出大量日值LAI=0.05，当关闭作物 CROP_OPTION = 0时，输出的日值正常了，请问这是什么原因，是我编译没有成功吗？请求各位大佬解答，谢谢。

**Comment by cenlinhe** [comment]
One possible reason is that your current simulation period does not reach the planting date. Please double check the planting date setup for your crop simulations. The crop only grows after planting date.

**Reply by 12wlj** [reply]
我作物模拟的种植日期设置的没问题，模拟的作物产量是有的，但是LAI值就不正确，后面一直是0.05，是编译时候作物这块没有编译好吗

**Reply by cenlinhe** [reply]
@CharlesZheZhang

**Reply by CharlesZheZhang** [reply]
The crop model sets a minimum value for LAI be 0.05. If the LAI output is always 0.05, this suggests a very low LAI (photosynthesis). This can be confirmed with low leaf mass (LFMASS) and GRAIN output as well. 
Please indicates the simulation period (planting/harvest date), single-point or 2D regional modeling. One possible cause is the model requires forcing data in UTC time zone, please check if running single-point simulation.

**Reply by ar1128** [reply]
> Please indicates the simulation period (planting/harvest date), single-point or 2D regional modeling. One possible cause is the model requires forcing data in UTC time zone, please check if running single-point simulation.

Hi, I was curious on how to run the crop model for a single point simulation.

I have been running the online model (coupled with WRF) till now, and wanted to run single point simulations for model validation.

**Reply by cenlinhe** [reply]
Here is the tutorial for single point offline Noah-MP simulation: https://github.com/NCAR/hrldas/blob/master/tutorial/Note1_Single_Point.ipynb
In addition, for crop simulation, you may also need to extract your planting, harvesting, and total GDD values from geo_em file to add them to the generated HRLDAS_setup file. The pre-processing scripts in the current HRLDAS will not extract crop-related variables. 
@CharlesZheZhang We need to improve the pre-processing package in the future for this.
